packages/kotti-ui/source/next/kotti-field/KtField.vue

```diff
@@ -282,20 +297,27 @@ export default defineComponent({
 
 			&__tooltip {
 				position: absolute;
-				bottom: 20px;
-				display: none;
-				padding: 0.25em;
-				color: var(--support-info);
+				z-index: 1; // fix weird layout issues with el-select
+				width: max-content;
+				max-width: 12rem;
+				max-height: 7em;
+				padding: 0.75em 1em;
+				overflow: scroll;
+				color: var(--gray-10);
 				cursor: none;
-				background-color: var(--ui-01);
+				background-color: var(--gray-80);
```

Comment:
try ui-04, it's the closest to that, and thn we stick to the tokens for the background
check this out if you didn't know about it already
https://3yourmind.github.io/kotti/tokens/colors